[2025-03-31T11:30:30.967+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: test_dag2.move_to_transformation manual__2025-03-31T10:30:22.437205+00:00 [queued]>
[2025-03-31T11:30:30.994+0100] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: test_dag2.move_to_transformation manual__2025-03-31T10:30:22.437205+00:00 [queued]>
[2025-03-31T11:30:30.995+0100] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-03-31T11:30:31.045+0100] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): move_to_transformation> on 2025-03-31 10:30:22.437205+00:00
[2025-03-31T11:30:31.058+0100] {standard_task_runner.py:60} INFO - Started process 5809 to run task
[2025-03-31T11:30:31.067+0100] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'test_dag2', 'move_to_transformation', 'manual__2025-03-31T10:30:22.437205+00:00', '--job-id', '194', '--raw', '--subdir', 'DAGS_FOLDER/test_dag2.py', '--cfg-path', '/tmp/tmpiav4_thb']
[2025-03-31T11:30:31.077+0100] {standard_task_runner.py:88} INFO - Job 194: Subtask move_to_transformation
[2025-03-31T11:30:31.260+0100] {task_command.py:423} INFO - Running <TaskInstance: test_dag2.move_to_transformation manual__2025-03-31T10:30:22.437205+00:00 [running]> on host LAPTOP-A4DHQTS9.
[2025-03-31T11:30:31.611+0100] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='test_dag2' AIRFLOW_CTX_TASK_ID='move_to_transformation' AIRFLOW_CTX_EXECUTION_DATE='2025-03-31T10:30:22.437205+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-03-31T10:30:22.437205+00:00'
[2025-03-31T11:30:31.682+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-31T11:30:31.715+0100] {sql.py:450} INFO - Running statement: 
        SELECT column_name FROM information_schema.columns 
        WHERE table_schema = 'transformation' 
        AND table_name = 'trans_prog3'
        AND column_name <> 'uid';
    , parameters: None
[2025-03-31T11:30:31.743+0100] {sql.py:459} INFO - Rows affected: 3
[2025-03-31T11:30:31.790+0100] {base.py:83} INFO - Using connection ID 'destination_db' for task execution.
[2025-03-31T11:30:31.824+0100] {sql.py:450} INFO - Running statement: 
        INSERT INTO transformation.trans_prog3
        SELECT * FROM staging.stg_prog3
        ON CONFLICT (uid) DO UPDATE 
        SET "lastupdated" = EXCLUDED."lastupdated", "Area (H)" = EXCLUDED."Area (H)", "farmers" = EXCLUDED."farmers"
        WHERE transformation.trans_prog3.lastupdated < EXCLUDED.lastupdated;
    , parameters: None
[2025-03-31T11:30:31.828+0100] {sql.py:459} INFO - Rows affected: 1
[2025-03-31T11:30:31.830+0100] {python.py:201} INFO - Done. Returned value was: None
[2025-03-31T11:30:31.928+0100] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=test_dag2, task_id=move_to_transformation, execution_date=20250331T103022, start_date=20250331T103030, end_date=20250331T103031
[2025-03-31T11:30:32.027+0100] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-03-31T11:30:32.107+0100] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
